### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by austincunningham**
## Overview

Add licence to CRO, same as integreatly-operator

## Checklist
- [ ] This PR includes a change to an instance type, I have used script `hack/<provider>/supported_types.sh` and attached the result below

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (laurafitzgerald) [inline LICENSE:189]: Should this be updated?

Comment (austincunningham) [reply LICENSE:189]: good point , integreatly-operator is the same with the place holder still in place :(

looking for an example of where we have this done
aerogear/offix has the line removed https://github.com/aerogear/offix/blob/master/LICENSE#L187-L189
aerogear/aerogear-js-sdk has 
```
Copyright 2018 Aerogear
``` 
https://github.com/aerogear/aerogear-js-sdk/blob/master/LICENSE#L189

Guess we could stick in
```
Copyright 2019 Redhat
```

File: LICENSE [hunk 2]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2019 Redhat
```
Comment (matskiv) [inline LICENSE:189]:
```suggestion
   Copyright 2019 Red Hat, Inc.
```

Comment (matskiv) [reply LICENSE:189]: 🤔 should it be `2021` ?

Comment (austincunningham) [reply LICENSE]: Think it should be from when the repo was created . Thats a best guess.

**Pull request merged**